public/javascript/pump.js

```diff
@@ -222,20 +222,9 @@ if (!window.Pump) {
     // send them here and I'll figure it out.
 
     Pump.error = function(err) {
-        var msg;
+        var msg = Pump.getError(err);
 
-        if (_.isString(err)) {
-            msg = err;
-        } else if (_.isObject(err)) {
-            msg = err.message;
-            if (err.stack) {
-                console.log(err.stack);
-            }
-        } else {
-            msg = "An error occurred.";
-        }
-
-        console.log(msg);
+        Pump.debug(msg);
```

Comment:
I think probably we should send this to the error console instead (i.e. `console.error`). I'm thinking that's probably the right thing even if `debugClient` isn't enabled; what do you think?

Reply:
I think is not necessary because the console log can be for example "invalid user credentials" don't show a different message of alert banner and in `getError` show the `console.log` in case it has a stack object that I think is more relevant